bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -379,10 +397,17 @@ public void readComplete(int rc, LedgerHandle lh,
                 final long dataLength = data.length;
                 numEntriesRead.inc();
                 numBytesRead.registerSuccessfulValue(dataLength);
+                ++replicatedEntriesNum;
+
                 ByteBufList toSend = lh.getDigestManager()
                         .computeDigestAndPackageForSending(entryId,
                                 lh.getLastAddConfirmed(), entry.getLength(),
                                 Unpooled.wrappedBuffer(data, 0, data.length));
+                int toSendSize = toSend.readableBytes();
+
+                totalReplicatedEntriesSize += toSendSize;
+                averageEntrySize = (int) ((totalReplicatedEntriesSize / replicatedEntriesNum));
```

Comment:
does the `avg = avg * ratio + (1 - ratio) * newSize` method to calculate averageEntrySize doesn't work well?